deployments/examples/ocis_wopi/.env

```diff
@@ -159,7 +160,7 @@ COLLABORA_ADMIN_PASSWORD=
 
 ## Uppy Companion Settings ##
 # the leading colon is required to enable the service
-COMPANION=:companion.yml
+#COMPANION=:companion.yml
```

Comment:
why do you remove the space? it looks weird.

Reply:
Because it differenciates from text comments and commented keys.